file/json/aws_lambda/index.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+let s3 = new AWS.S3();
+
+let bucketName = "lambda-function-records";
+let bucketParams = { Bucket: bucketName };
+
+let limitNumberOfFileToUpdateInOneRun = 2;
+
+exports.handler = (request, context, callback) => {
+    update(request.state, request.secrets, callback);
+};
+
+async function update(state, secrets, callback) {
+    let response = initializeResponse(state);
+
+    let modifiedFiles = [];
+
+    await s3.listObjects(bucketParams).promise().then(result => {
+        for (let index = 0; index < result.Contents.length; index++) {
```

Comment:
I didn't use forEach here because I was not able to use break statement into forEach.

Reply:
what's about `for of`?